spec/src/main/asciidoc/app-programming-model.adoc

```diff
@@ -155,9 +155,67 @@ MAY stop the deployment of an application if Metadata is missing.
 
 [[annotated-naming-convention]]
 ==== Annotated Naming Convention
-Annotated metrics are registered into the _application_ `MetricRegistry` with the name based on the annotation's `name` and `absolute` fields.
+Annotated metrics are registered into the _application_ `MetricRegistry` with the name computed using the rules in the following tables.
 
-.Example of annotated metric names
+If the metric annotation is placed on a method or field:
+|===
+| | `name` is specified | `name` is not specified
+| `absolute=true` | Value of the `name` argument | Name of the annotated element
+| `absolute=false` | `<canonical-name-of-declaring-class>.<value-of-name-argument>` | `<canonical-name-of-declaring-class>.<name-of-element>`
+|===
+
+If the metric annotation is placed on a class, then for each method, the metric name will be:
```

Comment:
```suggestion
If the metric annotation is placed on a class, then for each constructor or method, the metric name will be:
```

Reply:
how about `method (including constructors)`? To just make it clear that by methods we mean constructors as well, which are just a special kind of methods

Reply:
happy with that